src/steps/issues.ts

```diff
@@ -65,81 +65,86 @@ export async function fetchIssues({
             );
           }
-        }
-
-        if (issue.fields.reporter && issue.fields.reporter.accountId) {
-          const reporterUserKey = generateEntityKey(
-            USER_ENTITY_TYPE,
-            issue.fields.reporter.accountId,
-          );
-          const reporterEntity = (await jobState.findEntity(
-            reporterUserKey,
-          )) as UserEntity;
 
-          if (reporterEntity) {
-            await jobState.addRelationship(
-              createDirectRelationship({
-                _class: RelationshipClass.REPORTED,
-                from: reporterEntity,
-                to: issueEntity,
-              }),
+          if (issue.fields.creator && issue.fields.creator.accountId) {
+            const creatorUserKey = generateEntityKey(
+              USER_ENTITY_TYPE,
+              issue.fields.creator.accountId,
             );
-          } else {
-            logger.warn(
-              { reporterUserKey, issueName: issueEntity.name },
-              'Reported user is no longer in this Jira instance. Not creating user_reported_issue relationship.',
+            const creatorEntity = (await jobState.findEntity(
+              creatorUserKey,
+            )) as UserEntity;
+
+            if (creatorEntity) {
+              await jobState.addRelationship(
+                createDirectRelationship({
+                  _class: RelationshipClass.CREATED,
+                  from: creatorEntity,
+                  to: issueEntity,
+                }),
+              );
+            } else {
+              logger.warn(
+                { creatorUserKey, issueName: issueEntity.name },
+                'Created user is no longer in this Jira instance. Not creating user_created_issue relationship.',
+              );
+            }
+          }
+
+          if (issue.fields.reporter && issue.fields.reporter.accountId) {
+            const reporterUserKey = generateEntityKey(
+              USER_ENTITY_TYPE,
+              issue.fields.reporter.accountId,
             );
+            const reporterEntity = (await jobState.findEntity(
+              reporterUserKey,
+            )) as UserEntity;
+
+            if (reporterEntity) {
+              await jobState.addRelationship(
+                createDirectRelationship({
+                  _class: RelationshipClass.REPORTED,
+                  from: reporterEntity,
+                  to: issueEntity,
+                }),
+              );
+            } else {
+              logger.warn(
+                { reporterUserKey, issueName: issueEntity.name },
+                'Reported user is no longer in this Jira instance. Not creating user_reported_issue relationship.',
+              );
+            }
           }
+        } catch (err) {
+          //if a single issue has an error in processing, just log it and continue 'cause we got a lotta things to do
+          logger.warn(err, `Error encountered on issue ${issue.id}`);
```

Comment:
This is a good use of `logger.warn` because we should monitor these to discover when there are a significant number of failures.
```suggestion
          logger.warn({ err, issueId: issue.id }, 'Error encountered processing issue');
```

Reply:
Done.